tests/float/complex1.py

```diff
@@ -4,9 +4,17 @@
 print(complex(1))
 print(complex(1.2))
 print(complex(1.2j))
+print(complex("j"))
```

Comment:
Should we add cases with upper-case J as well?

Reply:
added